eng/CHECKENFORCER

```diff
@@ -1,41 +0,0 @@
-format: v0.1-alpha
```

Comment:
@benbp did you have any plans to use this configuration with the new checkenforcer? If not then we should probably remove this from the other repos too.

Reply:
No plans. I have an item on my todo to remove the other markdown files from the repos.

Reply:
Ok make sure your todo list translates to github issues so we don't lose them.

Reply:
@benbp I can go ahead and get rid of all the `CHECKENFORCER` files, given I already work on this.

Can I maybe also somehow assist with merging this?
https://github.com/Azure/azure-dev/pull/981

Just hoping here to wrap-up this removal work and not leave anything dangling.